src/gui/src/CapriceVKeyboard.cpp

```diff
@@ -119,21 +119,22 @@ namespace wGui {
           keycode += MOD_EMU_KEY;
           // Lookup the SDL key corresponding to this emulator command
           for (dword n = 0; n < KBD_MAX_ENTRIES; n++) {
-            if(keycode == kbd_layout[CPC.kbd_layout][n][0]) {
-              key.key.keysym.sym = static_cast<SDLKey>(kbd_layout[CPC.kbd_layout][n][1] & 0xffff);
-              key.key.keysym.mod = static_cast<SDLMod>(kbd_layout[CPC.kbd_layout][n][1] >> 16);
+            if(keycode == kbd_layout[n][0]) {
+              key.key.keysym.sym = static_cast<SDLKey>(kbd_layout[n][1] & 0xffff);
+              key.key.keysym.mod = static_cast<SDLMod>(kbd_layout[n][1] >> 16);
             }
           }
         } else {
-          key.key.keysym.sym = static_cast<SDLKey>(kbd_layout[CPC.kbd_layout][keycode][1] & 0xffff);
-          key.key.keysym.mod = static_cast<SDLMod>(kbd_layout[CPC.kbd_layout][keycode][1] >> 16);
+          key.key.keysym.sym = static_cast<SDLKey>(kbd_layout[keycode][1] & 0xffff);
+          key.key.keysym.mod = static_cast<SDLMod>(kbd_layout[keycode][1] >> 16);
         }
         escaped = false;
         cap32_cmd = false;
       } else {
+// TODO Fix this !
```

Comment:
I have the habit of putting TODO(cpitrat) (so you could use TODO(sebhz)).
Not mandatory but this may help you find your TODOs in the code more easily.